src/openms/source/FORMAT/GNPSMetaValueFile.cpp

```diff
@@ -43,16 +43,21 @@
 namespace OpenMS
 {   
     /**
-    @brief Generates a meta value required for GNPS FBMN, as defined here: https://ccms-ucsd.github.io/GNPSDocumentation/metadata/
+    @brief Generates a meta value table required for GNPS FBMN, as defined here: https://ccms-ucsd.github.io/GNPSDocumentation/metadata/
     */
-    void GNPSMetaValueFile::store(const StringList& mzml_file_paths, const String& output_file)
-    {
+    void GNPSMetaValueFile::store(const ConsensusMap& consensus_map, const String& output_file)
+    {   
+        StringList mzML_file_paths;
+        for (const auto& header: consensus_map.getColumnHeaders())
```

Comment:
can you check if https://github.com/OpenMS/OpenMS/blob/a686a995efc5d2f38519666039d856566a571654/src/openms/source/KERNEL/ConsensusMap.cpp#L570-L577
does the same?
then this would only be one line of code

Reply:
of course, that's much better!